

for i=(-1,20) do
    try do
        var x = 1;
    catch ObjectException do
        print "ObjectException 1";
    end_try
    print "Hello!", x;
end_loop
